resources/[race]/race/modes/shooter.lua

```diff
@@ -587,7 +587,7 @@ function Shooter:launch()
 		setTimer(function() clientCall(g_Root, 'sh_initTimeBars') end,4500,1)
 	else
 		--if math.random(2) == 1 then clientCall(g_Root, 'showOnlyHealthBar', true) end -- Use this to hide names 50% of the time
-		clientCall(g_Root, 'showOnlyHealthBar', true) -- Use this to hide names 100% of the time
+		--clientCall(g_Root, 'showOnlyHealthBar', true) -- Use this to hide names 100% of the time
```

Comment:
I think the line should be removed entirely instead of being commented out, we can always revert this commit.